diff -p -up libgnomesu-1.0.0/src/services/pam.c~ libgnomesu-1.0.0/src/services/pam.c
--- libgnomesu-1.0.0/src/services/pam.c~	2005-02-13 14:34:16.000000000 +0100
+++ libgnomesu-1.0.0/src/services/pam.c	2009-11-11 09:39:51.000000000 +0100
@@ -64,7 +64,7 @@ bomb (GnomesuAuthDialog *auth, gchar *fo
 	dialog = gtk_message_dialog_new ((GtkWindow *) auth,
 		GTK_DIALOG_MODAL,
 		GTK_MESSAGE_ERROR, GTK_BUTTONS_OK,
-		msg);
+					 "%s", msg);
 	gtk_window_set_resizable (GTK_WINDOW (dialog), FALSE);
 	gtk_dialog_run (GTK_DIALOG (dialog));
 	gtk_widget_destroy (dialog);
diff -p -up libgnomesu-1.0.0/src/services/su.c~ libgnomesu-1.0.0/src/services/su.c
--- libgnomesu-1.0.0/src/services/su.c~	2005-02-13 14:34:20.000000000 +0100
+++ libgnomesu-1.0.0/src/services/su.c	2009-11-11 09:40:06.000000000 +0100
@@ -60,7 +60,7 @@ bomb (GnomesuAuthDialog *auth, gchar *fo
 	dialog = gtk_message_dialog_new ((GtkWindow *) auth,
 		GTK_DIALOG_MODAL,
 		GTK_MESSAGE_ERROR, GTK_BUTTONS_OK,
-		msg);
+		"%s", msg);
 	gtk_window_set_resizable (GTK_WINDOW (dialog), FALSE);
 	gtk_dialog_run (GTK_DIALOG (dialog));
 	gtk_widget_destroy (dialog);
